### ISSUE TITLE
"Forecast" feature

### ISSUE THREAD

**Issue by olarosling** [Gapminder/vizabi#1072]
Add to the state a universal property across all vizabi tools, which  disables future data from being loaded, unless the state permits it. The same state properties are also used to achieve this kind of chart with stripes in the future>
![image](https://cloud.githubusercontent.com/assets/496398/11506211/8489dc42-984e-11e5-9c0c-e01d16071787.png)

Avoid showing future data, unless explicitly set to a year in the future.

state.future_starts = "next" or a specific year
state.future_ends = "*" or a specific year,.. which is the end of the future data automatically pulled.

The future_ends  property determines how long the timesldier will extend into the future (if there is data) ,
The period between future_starts, and future_ends will be displayed differently in most vizabi tools. with stripes on the the data-markers.

(This enables quick extension of the time-slider to either include or not include projections. ) 
Clicking the future button, should trigger a new data query to append the missing years (or until we optimize: append all years up to the future_ends property)

**Comment by jheeffer** [comment]
Nice feature, definitely good to have, I think @angieskazka already did some trials for the vizulation on this right?

However, technically, is this the right way to do this? It seems to be meta-data rather than program state. I feel a separate column or a meta-data file should describe if certain indicators-values/years in the data are predictions or not. Then Vizabi can show them that way.

The way you propose it you:
1. Have to set what year/values are predictions for every tool which uses the same data (redundant)
2. Developers rather than data-providers/owners 'decide' if data is a forecast or not (not the right people).

So: Wouldn't it be better to include this information in the (meta)data rather than in the program state and make Vizabi able to distinguish measured and forecast data?

**Comment by angieskazka** [comment]
no, i didn't do any progress yet except that i found this tutorial: http://tutorials.jenkov.com/svg/fill-patterns.html
